<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>TableView</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>plainTextEdit</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_openFile"/>
   <addaction name="act_ShowData"/>
   <addaction name="separator"/>
   <addaction name="act_AddRow"/>
   <addaction name="act_InsertRow"/>
   <addaction name="act_deleteRow"/>
   <addaction name="separator"/>
   <addaction name="act_Center"/>
   <addaction name="act_left"/>
   <addaction name="act_Right"/>
   <addaction name="act_Borld"/>
   <addaction name="separator"/>
   <addaction name="act_qiuck"/>
  </widget>
  <action name="act_openFile">
   <property name="icon">
    <iconset resource="res2.qrc">
     <normaloff>:/icon/Text_Icon2/combination.png</normaloff>:/icon/Text_Icon2/combination.png</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="act_ShowData">
   <property name="icon">
    <iconset resource="res2.qrc">
     <normaloff>:/icon/Text_Icon2/sidebar.png</normaloff>:/icon/Text_Icon2/sidebar.png</iconset>
   </property>
   <property name="text">
    <string>数据浏览</string>
   </property>
  </action>
  <action name="act_AddRow">
   <property name="text">
    <string>添加行</string>
   </property>
  </action>
  <action name="act_InsertRow">
   <property name="text">
    <string>插入行</string>
   </property>
  </action>
  <action name="act_deleteRow">
   <property name="text">
    <string>删除行</string>
   </property>
  </action>
  <action name="act_Center">
   <property name="icon">
    <iconset resource="res2.qrc">
     <normaloff>:/icon/TextIcon/002-alignment.png</normaloff>:/icon/TextIcon/002-alignment.png</iconset>
   </property>
   <property name="text">
    <string>居中</string>
   </property>
  </action>
  <action name="act_left">
   <property name="icon">
    <iconset resource="res2.qrc">
     <normaloff>:/icon/TextIcon/005-rightalignment.png</normaloff>:/icon/TextIcon/005-rightalignment.png</iconset>
   </property>
   <property name="text">
    <string>居左</string>
   </property>
  </action>
  <action name="act_Right">
   <property name="icon">
    <iconset resource="res2.qrc">
     <normaloff>:/icon/TextIcon/005-rightalignment.png</normaloff>:/icon/TextIcon/005-rightalignment.png</iconset>
   </property>
   <property name="text">
    <string>居右</string>
   </property>
  </action>
  <action name="act_Borld">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res2.qrc">
     <normaloff>:/icon/TextIcon/009-bold.png</normaloff>:/icon/TextIcon/009-bold.png</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
  </action>
  <action name="act_qiuck">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>act_qiuck</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
